Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp

```diff
@@ -36,17 +36,16 @@
 
 ControllerInterface g_controller_interface;
 
-//
-// Init
-//
-// Detect devices and inputs outputs / will make refresh function later
-//
 void ControllerInterface::Initialize(const WindowSystemInfo& wsi)
 {
   if (m_is_init)
     return;
 
   m_wsi = wsi;
+
+  // Allow backends to add devices as soon as they are initialized.
```

Comment:
Note to self: SDL _may_ end up adding devices during `ciface::SDL::Init()` and not during `RefreshDevices()`